dags/cluster/tasks/business_logic/cluster_acteurs_suggestions/prepare.py

```diff
@@ -1,17 +1,18 @@
+import logging
+
 import pandas as pd
 from cluster.config.constants import COL_PARENT_DATA_NEW
 from utils import logging_utils as log
 from utils.data_serialize_reconstruct import data_serialize
 from utils.django import django_setup_full
 
-django_setup_full()
+logger = logging.getLogger(__name__)
 
+django_setup_full()
 
-def cluster_acteurs_suggestions_prepare(
-    df_clusters: pd.DataFrame,
-) -> list[dict]:
-    """Generate suggestions from clusters"""
 
+def cluster_changes_get(cluster: pd.DataFrame) -> list[dict]:
```

Comment:
```suggestion
def cluster_changes_get(cluster: pd.DataFrame) -> list[dict]:
```

Suggestion (nommage), comme on renomme : `get_changes_from`, réduit la répétition et est explicité quant au fait qu'on extrait les changements du cluster. 

Ou alors `generate_changes_from`, cf commentaire ci-dessous.